src/engine/BMGame.php

```diff
@@ -437,17 +342,17 @@ protected function update_game_state_start_game() {
         $this->nRecentPasses = 0;
         $this->autopassArray = array_fill(0, $this->nPlayers, FALSE);
         $this->fireOvershootingArray = array_fill(0, $this->nPlayers, FALSE);
-        $this->gameScoreArrayArray = array_fill(0, $this->nPlayers, array(0, 0, 0));
```

Comment:
Nit: it seems like this all zeroes array was simply a bug which was harmlessly overwritten by `do_next_step_apply_handicaps()` to have the correct keys.  Maybe that means that one or the other of these functions doesn't *need* to zero-fill the array?